src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -3,89 +3,39 @@
   </PropertyGroup>
+  
   <PropertyGroup>
     <CopyRazorGenerateFilesToPublishDirectory>true</CopyRazorGenerateFilesToPublishDirectory>
   </PropertyGroup>
+  
   <ItemGroup>
+    <ProjectReference Include="../Umbraco.Persistence.SqlCe/Umbraco.Persistence.SqlCe.csproj" Condition="'$(OS)' == 'Windows_NT'" />
     <ProjectReference Include="../Umbraco.PublishedCache.NuCache/Umbraco.PublishedCache.NuCache.csproj" />
     <ProjectReference Include="../Umbraco.Web.BackOffice/Umbraco.Web.BackOffice.csproj" />
     <ProjectReference Include="../Umbraco.Web.Common/Umbraco.Web.Common.csproj" />
     <ProjectReference Include="../Umbraco.Web.Website/Umbraco.Web.Website.csproj" />
-    <ProjectReference Include="../Umbraco.Persistence.SqlCe/Umbraco.Persistence.SqlCe.csproj" Condition="'$(OS)' == 'Windows_NT'" />
   </ItemGroup>
 
-
   <ItemGroup>
     <PackageReference Include="Microsoft.ICU.ICU4C.Runtime" Version="68.2.0.9" />
-
     <RuntimeHostConfigurationOption Condition="$(RuntimeIdentifier.StartsWith('linux')) Or $(RuntimeIdentifier.StartsWith('win')) Or ('$(RuntimeIdentifier)' == '' And !$([MSBuild]::IsOSPlatform('osx')))" Include="System.Globalization.AppLocalIcu" Value="68.2.0.9" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="App_Plugins" />
-    <Folder Include="Views" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Compile Remove="umbraco/Data/**" />
-    <Compile Remove="umbraco/logs/**" />
-    <Compile Remove="umbraco/mediacache/**" />
-    <Compile Remove="wwwroot/umbraco/**" />
-    <Compile Remove="App_Data/**" />
-    <Compile Remove="Controllers\**" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Remove="App_Data/**" />
-    <EmbeddedResource Remove="umbraco/Data/**" />
-    <EmbeddedResource Remove="umbraco/logs/**" />
-    <EmbeddedResource Remove="umbraco/mediacache/**" />
-    <EmbeddedResource Remove="Controllers\**" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <Content Remove="App_Data/**" />
-    <Content Remove="umbraco/Data/**" />
-    <Content Remove="umbraco/logs/**" />
-    <Content Remove="umbraco/mediacache/**" />
-    <Content Remove="umbraco\UmbracoWebsite\NotFound.cshtml" />
-    <Content Remove="wwwroot/Web.config" />
-    <Content Remove="Controllers\**" />
-  </ItemGroup>
-
-
-  <ItemGroup>
-    <None Include="config/**/*.*">
-      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-      <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-    </None>
-    <None Include="umbraco/**/*.*">
-      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
-      <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-    </None>
-    <None Remove="App_Data/**" />
-    <None Remove="umbraco/Data/**" />
-    <None Remove="umbraco/logs/**" />
-    <None Remove="umbraco/mediacache/**" />
-    <None Include="umbraco/UmbracoWebsite/NoNodes.cshtml" />
-    <None Remove="Controllers\**" />
-  </ItemGroup>
-
-  <!-- We don't want to include the generated files, they will throw a lot of errors -->
-
```

Comment:
In our staticassets target file we have the following:
```

        <DefaultItemExcludes>$(DefaultItemExcludes);App_Plugins\**;</DefaultItemExcludes>
        <DefaultItemExcludes>$(DefaultItemExcludes);umbraco\Data\**;</DefaultItemExcludes>
        <DefaultItemExcludes>$(DefaultItemExcludes);umbraco\Logs\**;</DefaultItemExcludes>
        <DefaultItemExcludes>$(DefaultItemExcludes);umbraco\mediacache\**;</DefaultItemExcludes>
        <DefaultItemExcludes>$(DefaultItemExcludes);wwwroot\media\**;</DefaultItemExcludes>
```

Not sure if `/` vs `\` is a problem for linux.

Reply:
I've updated this to import the MSBuild properties from the `Umbraco.Cms.StaticAssets.props` file that's included in the NuGet package and only added an additional exclude for `wwwroot/umbraco/**` (this will be generated by the BelleBuild target anyway):

https://github.com/umbraco/Umbraco-CMS/blob/8d82f86c29ec7ae5ab865cbc414baee39c484b69/src/Umbraco.Web.UI/Umbraco.Web.UI.csproj#L8-L11

And the [MS Docs use forward slashes](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#defaultitemexcludes), so I've ensure to use the same here 👍🏻